### PR TITLE
Adding section to avoid issue with Ci/CD

### PR DESCRIPTION
**Pull request by theburi**
## Description

This is a change to help our customers avoid confusion with CI/CD configuration.

## When should this change go live?

- [X] This change is not yet live and should not be merged until {ADD_DATE} (apply `hold` label or convert to draft PR)?
- [ ] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [ ] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [ ] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [X] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [ ] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

File: docs/self-managed/setup/install.md [hunk 1]
```diff
@@ -104,6 +155,8 @@ Installing all the components in a cluster requires all Docker images to be down
 
 For air-gapped environments, refer to [installing in an air-gapped environment](/self-managed/setup/guides/air-gapped-installation.md).
 
+This command does not install Identity.
```
Comment (jessesimpson36) [inline docs/self-managed/setup/install.md:158]: I feel like this is not accurate.  Identity by default should be enabled.

Did you try going through this guide, and found that the identity pod is not getting created? if so, that might be a bug.

**Review by jessesimpson36 — CHANGES_REQUESTED** [review]

File: docs/self-managed/setup/install.md (hunk 1 above)
Comment (theburi) [reply docs/self-managed/setup/install.md:158]: You are right. It was an artifact from my first iteration.

**Review by jessesimpson36 — APPROVED** [review]

**Review by aabouzaid — DISMISSED** [review]

**Comment by akeller** [comment]
@christinaausley, in addition to a review for style/grammar, do you have guidance on the secrets in code snippets? This might be something to add to the style guide.

**Comment by christinaausley** [comment]
> secrets in code snippets

I don't have a strong preference for this, so long as we are consistent. The hyphens and simple naming conventions work well -- will add to the style guide.

**Review by christinaausley — APPROVED** [review]

**Pull request merged**